app/src/main/res/layout/fragment_leave_review.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="16dp"
+    android:id="@+id/leave_review_dialog_fragment"
+    android:fitsSystemWindows="true">
+
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/leave_review_dialog_fragment"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:fitsSystemWindows="true"
    android:padding="16dp">

    <ScrollView
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <androidx.constraintlayout.widget.ConstraintLayout
            android:layout_width="match_parent"
            android:layout_height="wrap_content">

            <TextView
                android:id="@+id/leave_review_fragment_title"
                style="@style/TextAppearance.AppCompat.Title"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_margin="16dp"
                android:text="@string/leave_a_review"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toTopOf="parent" />

            <RatingBar
                android:id="@+id/leave_review_fragment_rating"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_marginTop="16dp"
                android:stepSize="1"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toBottomOf="@id/leave_review_fragment_title" />

            <EditText
                android:id="@+id/leave_review_fragment_feedback_text"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_margin="16dp"
                android:autofillHints="@string/leave_a_review_hint"
                android:hint="@string/leave_a_review_hint"
                android:inputType="textMultiLine"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toBottomOf="@id/leave_review_fragment_rating" />

            <Button
                android:id="@+id/leave_review_fragment_cancel_button"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_margin="16dp"
                android:text="@string/EditProfileActivity_Cancel"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toBottomOf="@id/leave_review_fragment_feedback_text" />


            <Button
                android:id="@+id/leave_review_fragment_save_button"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_margin="16dp"
                android:text="@string/EditProfileActivity_Save"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintTop_toBottomOf="@id/leave_review_fragment_feedback_text" />
        </androidx.constraintlayout.widget.ConstraintLayout>
    </ScrollView>
</androidx.constraintlayout.widget.ConstraintLayout>
```

![image](https://user-images.githubusercontent.com/25907972/117588630-d8d97100-b124-11eb-85b1-a7cdcc03f2e9.png)
It is a small detail, but if someone has a lot to say about the event, they will not be able to click on the button to leave a rating, I propose you to add a scrollview, Insinde your constraintlayout, so you can scroll and press on the button ;)

Reply:
Also do not accept the code suggestion, I did not select all the lines while making the comment.
But you can copy paste it and commit if you like my idea.